File: 03-11-listas/Main.java
import adts.list.LinkedList;
import adts.list.List;

public class Main {
   public static void main(String[] args) {
      List<String> lStr = new LinkedList<>();

      System.out.println(lStr.exists("hola"));

      lStr.add("hola");

      System.out.println(lStr.exists("hola"));
   }

}

File: 03-11-listas/main.cpp
#include "./adts/list/linked_list.cpp"
#include <string>
#include <iostream>

int main()
{

   list<std::string> *lStr = new linked_list<std::string>();
   std::cout << lStr->exists("hola") << std::endl;

   lStr->add("hola");

   std::cout << lStr->exists("hola") << std::endl;

   return 0;
}
